landkreise/get-heinsberg-kreis.py

```diff
@@ -0,0 +1,29 @@
+import re
+import requests
+from bs4 import BeautifulSoup
+
+import scrape
+import helper
+
+main_url = "https://www.kreis-heinsberg.de/aktuelles/aktuelles/?pid=5142"
+community_id = "05370"
+
+case_regex = r"(Infektionen liegen bei|gibt es im Kreis Heinsberg)[^0-9]*([0-9]+)"
+date_regex = r"Kreis Heinsberg[\. ]*\(([0-9\.]+)\)"
+date_format = '%d.%m.%Y'
+
+req = requests.get(main_url)
+req.encoding = 'iso-8859-1'
+bs = BeautifulSoup(req.content, "html.parser")
```

Comment:
If I see it correctly, those lines arent needed, since the BeautifulSoup handling is done in scrape.py.

By the way, did you know, you can use the function get_status from helper.py to extract the status date? It tries a variety of different formats and thus is more robust.